### DISCUSSION TITLE
New deploys are getting error

### DISCUSSION THREAD

**Discussion by hhokuno** [microsoft/sample-app-aoai-chatGPT#1191] [Q&A] [answered]
Hello everybody,
Our new deploys of this web app are getting the error bellow when trying to chat with it. This error is very new, we were not getting it last week.

Please, can anybody help?

> Error
> AsyncClient.__init__() got an unexpected keyword argument 'proxies'
>

**Comment by chrismauck** [comment, marked as answer]
In requirements.txt, update openai to openai>=1.55.3. Fixed it for me.

**Reply by hhokuno** [reply]
Hello, @chrismauck . Thanks for the reply. Is there any chance of you to update it on this repository? I'm trying the manual deploy, but I'm not able. Probably doing something wrong, but I'm trying for weeks. I'm using the deploying from Azure OpenAI Studio. Now we have a delivery and I'm think its everything is missing to accomplish the achievement.

**Reply by chrismauck** [reply]
Hi @hhokuno I don't know that I have that ability. Unfortunately I'm also not working in Studio, but in a local codebase. So I'm unsure how to advise you to rectify on your end. If you're unable to access the requirements.txt at the root of the project then you may not be able to address.

**Reply by hhokuno** [reply]
Hello,
Someone created a branch with your solution, that I was able to point fron Azure and change the source of deployment. It worked.
I tried to aprove the pull request from that branch, but I dont know if I have the profile. Thank you for the solution.
